contributing_to_docs/doc_guidelines.adoc

```diff
@@ -128,62 +128,41 @@ ID. For example:
 == Managing authorization policies
 ----
 
-== Writing concepts
-A _concept_ contains information to support the tasks that users want to do and
-must not include task information like commands or numbered steps. In most
-cases, create your concepts as individual modules and include them in
-appropriate assemblies. Avoid using gerunds in concept titles. "About <concept>"
-is a common concept module title.
-
-For more information about writing concepts, see the
-link:https://redhat-documentation.github.io/modular-docs/[Red Hat modular docs reference guide].
+== Writing assemblies
+An _assembly_ is a collection of modules that describes how to accomplish a user story.
 
-== Writing tasks
-A _task_ contains the steps that users follow to complete a process. Tasks
-contain ordered steps and explicit commands. In most cases, create your tasks
-as individual modules and include them in appropriate assemblies.
+For more information about forming assemblies, see the
+link:https://redhat-documentation.github.io/modular-docs/[Red Hat modular docs reference guide] and the link:https://github.com/redhat-documentation/modular-docs/blob/master/modular-docs-manual/files/TEMPLATE_ASSEMBLY_a-collection-of-modules.adoc[assembly template].
```

Comment:
I would link to the raw version of the template: https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_ASSEMBLY_a-collection-of-modules.adoc

Reply:
Also for the first link, maybe go straight to the assembly section? https://redhat-documentation.github.io/modular-docs/#forming-assemblies